logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/UserSessionScope.kt

```diff
@@ -660,6 +662,14 @@ class UserSessionScope internal constructor(
         )
     }
 
+    val missingMetadataUpdateManager: MissingMetadataUpdateManager by lazy {
```

Comment:
This is currently never instantiated so it will never run?

Reply:
Why not run it after the incremental finishes like many other period tasks. Then we don't need any platform  specific code.

Reply:
Makes sense to me, and sounds better to reuse the mechanism we already have in place.